actions/js_09.md

```diff
@@ -0,0 +1,11 @@
+- date: 3/19/2020
+- sources: https://www.bloomberg.com/news/articles/2020-03-19/google-agrees-to-worker-council-in-europe-amid-recurring-discord
+- actions: union_representation
```

Comment:
we dont have `formed worker council` as a type of action

Reply:
legal_action?